### ISSUE TITLE
Possible error?

### ISSUE THREAD

**Issue by Emmanuel-Messulam** [aamini/evidential-deep-learning#15]
https://github.com/aamini/evidential-deep-learning/blob/d1d8e395fb083308d14fa92c5ce766e97b2a066a/evidential_deep_learning/losses/discrete.py#L25

Isn't it ` B = tf.reduce_sum(alpha*(S-alpha)/(S*S*(alpha+1)), axis=1, keepdims=True) ` from https://chcorbi.github.io/posts/2020/11/dirichlet-networks/#prior-networks ecuation (31)?
